### PR TITLE
App Dev Onboarding: First Batch

### PR DESCRIPTION
**Pull request by joemoore**
Partially done. Need to complete formatting.

### CODE REVIEW

**Comment by joemoore** [comment]
Currently
- Overview
- Balanced Teams
- Cloud Native Development
- Distributed teams (uses existing Remote Tips)

File: content/outcomes/application-development/cloud-native-development.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+In this article, you will learn to:
+
+- [Define one meaning of cloud native](#meaning)
+- [List three major outcomes of cloud native](#outcomes)
+- Explain the main cloud native concepts for ...
+	- [application architecture](#app-archi)
+	- [software life-cycle](#sw-life-cycle)
+	- [supporting infrastructure](#support-infra)
+	- [team culture](#culture)
+- [Identify the twelve factors of cloud native applications](#factors)
```
Comment (joemoore) [inline content/outcomes/application-development/cloud-native-development.md:22]: The section links are a great idea. We should do this for all of the articles.

File: content/outcomes/application-development/cloud-native-development.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+------
+
+## Define one meaning of cloud native {id=meaning}
+
+>  “Cloud is not a place; it’s a way of doing IT” - Michael Dell
+
+There are many definitions for cloud native. The definition we use in this article is an approach to building and running applications that exploit the advantages of the cloud computing delivery model. At its heart, cloud native is structuring teams, culture, and technology to utilize automation and architectures to manage complexity and unlock velocity.
+
+Cloud native development, often referred to as modern application development, defines how applications are created and deployed. It does not define where the deployment occurs. For example, public cloud, private cloud, hybrid cloud, multi-cloud.
```
Comment (joemoore) [inline content/outcomes/application-development/cloud-native-development.md:32]: >  For example, public cloud, private cloud, hybrid cloud, multi-cloud.

Let's link to definitions for these.

File: content/outcomes/application-development/cloud-native-development.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+2. Higher degree of automation, more determinism, less drudgery and less manual work. Infrastructure is self-healing, self-managed, and available as self-service.
+3. Increased non-functionalities for application landscape: improved reliability, portability, deep security, higher efficiency, and in-depth observability.
+
+## Explain the main cloud native concepts {id=concepts}
+
+Cloud native concepts are numerous. The following ones are the most common. They are sorted by their context:
+- [application architecture](#app-archi)
+- [software life-cycle](#sw-life-cycle)
+- [supporting infrastructure](#support-infra)
+- [team culture](#culture)
```
Comment (joemoore) [inline content/outcomes/application-development/cloud-native-development.md:48]: Are we asking the reader to explain these four cloud native concepts in general, or the 18 total concepts contained within?

File: content/outcomes/application-development/cloud-native-development.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+### Application architecture {id=app-archi}
+
+{{< table "table" >}}
+| Concept | Definition | 
+| ------- | ---------- |
+| Twelve-factor application | A set of twelve principles and practices that developers follow to construct applications optimized for modern cloud environments. Those twelve factors are often used to determine how well an application leverages cloud native technologies.
```
Comment (joemoore) [inline content/outcomes/application-development/cloud-native-development.md:55]:
```suggestion
| Twelve-factor application | A set of [twelve principles and practices](https://12factor.net/) that developers follow to construct applications optimized for modern cloud environments. Those twelve factors are often used to determine how well an application leverages cloud native technologies.
```

File: content/outcomes/application-development/cloud-native-development.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+| Cloud Native Patterns | A catalog of reusable solutions to recurring problems occurring in cloud native applications. Each pattern describes an architectural problem, its solution, and implied benefits, and drawbacks. |
+| Domain-Driven Design | A software development philosophy that encourages thinking in business domains at each step of the process: discovery, architecture, and design. Domain-Driven Design (DDD) comprises a variety of practices (for example, Event Storming) and concepts (for example, Bounded Context), while infusing a common "ubiquitous" language for better collaboration and alignment between business people and software engineers. DDD is not an all-or-nothing deal. You can apply the ideas from DDD as much or as little as you feel is beneficial to the project you're working on. |
```
Comment (joemoore) [inline content/outcomes/application-development/cloud-native-development.md:58]:
```suggestion
| Domain-Driven Design | A software development philosophy that encourages thinking in business domains at each step of the process: discovery, architecture, and design. [Domain-Driven Design (DDD)](https://en.wikipedia.org/wiki/Domain-driven_design) comprises a variety of practices (for example, Event Storming) and concepts (for example, Bounded Context), while infusing a common "ubiquitous" language for better collaboration and alignment between business people and software engineers. DDD is not an all-or-nothing deal. You can apply the ideas from DDD as much or as little as you feel is beneficial to the project you're working on. |
```

File: content/outcomes/application-development/cloud-native-development.md (hunk 1 above)
Comment (Maarc) [reply content/outcomes/application-development/cloud-native-development.md:24]: I am glad that you like it. I just updated the balanced-team article accordingly.

File: content/outcomes/application-development/cloud-native-development.md (hunk 2 above)
Comment (Maarc) [reply content/outcomes/application-development/cloud-native-development.md:34]: Done!

File: content/outcomes/application-development/cloud-native-development.md (hunk 3 above)
Comment (Maarc) [reply content/outcomes/application-development/cloud-native-development.md:50]: These four beasts are flagged as "context" for the concepts. Indeed, here the 18 concepts are in focus.

**Review by Maarc — DISMISSED** [review]
The two first articles and the overview are good to go for me.

**Review by Maarc — DISMISSED** [review]

File: content/outcomes/application-development/_index.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+title: Build Modern Applications
+weight: 1
+layout: single
+description: "Learn how we approach application development at VMware Tanzu Labs"
+team:
+  - VMware Tanzu Labs
+---
+ 
+## Why is this learning path important?
```
Comment (zmb3) [inline content/outcomes/application-development/_index.md:10]: Do we need to say learning path? Or is `Why is this important?` sufficient?

Also note, you've capitalized _Learning Path_ in the next section header but not in this one. We should be consistent.

File: content/outcomes/application-development/_index.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+* [Distributed, Remote, and Asynchronous Work](outcomes/application-development/remote/)
+* Pair Programming (coming soon)
+* Product Validation (coming soon)
+* Retrospectives (coming soon)
+* Test Driven Development (coming soon)
+* User Stories (coming soon)
+ 
+This learning path is also an adequate, on-the-clock preparation for getting the most out of an on-site or remote lab engagement.
+
+## Where can I share my feedback?
```
Comment (zmb3) [inline content/outcomes/application-development/_index.md:43]: Hmmm, if we want something like this it seems like we should solve it in a common place and not just on one of the outcome overview pages.

File: content/outcomes/application-development/balanced-teams.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+## Define balanced team for your team members {id=definition}
+> “A team management philosophy that has people with a variety of skills and perspectives that support each other towards a shared goal.” - *balancedteam.org*
+
+The typical balanced team mix for product development is a product manager, product designer, and engineer(s) dedicated to a single product development effort. This is in contrast to another common team structure where roles some roles, such as product management and design, are shared across many product efforts.
+
+When we work within the construct of a balanced team, we ensure that all these perspectives blend and inform each other so that we build products that are desirable, usable, feasible, and viable.
+
+Our product development teams are:
+- [Small enough to be fed by two pizzas](http://blog.idonethis.com/two-pizza-team/) as pioneered by Amazon
+- Co-located (In person or remotely)
```
Comment (zmb3) [inline content/outcomes/application-development/balanced-teams.md:31]: remotely co-located seems like an oxymoron to me - should we just remove this  bullet?

File: content/outcomes/application-development/_index.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
+---
+title: Build Modern Applications
```
Comment (zmb3) [inline content/outcomes/application-development/_index.md:2]: Recommend changing this to _Modern Application Development_ as the other outcome  titles are all nouns, not verbs.

File: content/outcomes/application-development/cloud-native-development.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+title: Cloud Native Development
+weight: 40
+layout: single
+team:
+  - VMware Tanzu Labs
+---
+
+Cloud Native is at the heart of modern application development endeavors. Used to depict engineering practices and state of the art applications, the term "cloud-native" is omnipresent and often misunderstood. There is a good chance that anyone you ask will provide a different definition.
+
+![Cloud Native Development](/images/outcomes/app-dev-onboarding/cloud-native-development.jpg)
```
Comment (zmb3) [inline content/outcomes/application-development/cloud-native-development.md:11]: This is not an image of cloud native development - it's just a picture of a cloud. I don't think it adds any value, but it does slow down the time it takes to load the page. Recommend removing.

File: themes/docsy/layouts/partials/outcomes-index.html [hunk 11]
```diff
@@ -1,14 +1,13 @@
...
-    {{ end }}
-    {{ end }}
+        <div class="col-lg-4 px-2 px-lg-2 mb-3">
+            <a class="linked-box td-box--gray-darkest px-4 py-2 h-100" href="{{ .RelPermalink }}">
+                <h4 class="link mt-3">{{- .LinkTitle -}}</h4>
+                <p>{{ .Description | default .Title | markdownify }}</p>
+            </a>
+        </div>
     {{ end }}
+    </div>
```
Comment (johnharris85) [inline themes/docsy/layouts/partials/outcomes-index.html:12]: What's the effect of / need for this change?

File: content/outcomes/application-development/cloud-native-development.md (hunk 10 above)
Comment (joemoore) [reply content/outcomes/application-development/cloud-native-development.md:11]: We like having some kind of engaging image at the beginning of these articles to break up the walls of text. Maybe we can find a different stock image.

File: content/outcomes/application-development/_index.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+ 
+Software engineering techniques have never evolved at such a quick pace as they are today. New frameworks, practices, and technologies keep emerging, while bringing new concepts and increased complexity of software. As a result, it feels hard to keep up with change and to make the right choices for your end-to-end modern application lifecycle.
+ 
+Beyond technology, this learning path also teaches the fundamentals of the 30+ year old [VMware Tanzu Labs](https://tanzu.vmware.com/labs) methodology on developing modern applications, including how to:
+ 
+- Build the right thing the first time.
+- Use Agile development to deliver the products your customers want.
+- Seed a new mindset. Quickly validate your concepts. Get them from idea to production in days, and not months.
+ 
+## Who is this Learning Path for?
```
Comment (joemoore) [inline content/outcomes/application-development/_index.md:20]:
```suggestion
## Who is this learning path for?
```

File: content/outcomes/application-development/_index.md (hunk 6 above)
Comment (joemoore) [reply content/outcomes/application-development/_index.md:10]: Good catch. Fixed.

Comment (joemoore) [inline content/outcomes/application-development/_index.md:10]:
```suggestion
## Why is this important?
```

File: content/outcomes/application-development/balanced-teams.md (hunk 8 above)
Comment (joemoore) [inline content/outcomes/application-development/balanced-teams.md:31]:
```suggestion
```

File: content/outcomes/application-development/_index.md (hunk 9 above)
Comment (joemoore) [inline content/outcomes/application-development/_index.md:2]:
```suggestion
title: Modern Application Development
```

File: themes/docsy/layouts/partials/outcomes-index.html (hunk 11 above)
Comment (Maarc) [reply themes/docsy/layouts/partials/outcomes-index.html:12]: It is fixing the layout for outcomes architectures. It was broken for more than 2 entries.

File: content/outcomes/application-development/_index.md (hunk 7 above)
Comment (Maarc) [reply content/outcomes/application-development/_index.md:43]: Good point! Then we probably should remove this.

On the internal [cookbooks](https://appmod-cookbook.apps.pcfone.io/), we added an "Improve this page" link heading to every page.

File: themes/docsy/layouts/partials/outcomes-index.html (hunk 11 above)
Comment (zmb3) [reply themes/docsy/layouts/partials/outcomes-index.html:12]: Jesse has a fix for this in #941 - recommend removing and rebasing on top of that fix.

Comment (BrianMMcClain) [reply themes/docsy/layouts/partials/outcomes-index.html:12]: +1, we just merged #941 as well, so let's remove this for now and rebase against that

Comment (joemoore) [reply themes/docsy/layouts/partials/outcomes-index.html]: Done

File: content/outcomes/application-development/_index.md (hunk 7 above)
Comment (joemoore) [reply content/outcomes/application-development/_index.md]: Done

File: content/outcomes/application-development/cloud-native-development.md (hunk 10 above)
Comment (joemoore) [reply content/outcomes/application-development/cloud-native-development.md]: Will iterate on images as we iterate on the rest of the content.

**Review by BrianMMcClain — APPROVED** [review]

**Comment by joemoore** [comment]
Thank you!

**Pull request merged**